x-pack/plugins/security_solution/cypress/e2e/investigations/alerts/expandable_flyout/alert_details_left_panel_correlations_tab.cy.ts

```diff
@@ -37,18 +47,40 @@ describe(
       openCorrelationsTab();
     });
 
-    it('should serialize its state to url', () => {
+    it('should render correlations details correctly', () => {
+      cy.log('should render the Insights header');
+
       cy.get(DOCUMENT_DETAILS_FLYOUT_INSIGHTS_TAB)
         .should('be.visible')
         .and('have.text', 'Insights');
 
+      cy.log('should render the inner tab switch');
       cy.get(DOCUMENT_DETAILS_FLYOUT_INSIGHTS_TAB_BUTTON_GROUP).should('be.visible');
 
+      cy.log('should render correlations tab activator / button');
       cy.get(DOCUMENT_DETAILS_FLYOUT_INSIGHTS_TAB_CORRELATIONS_BUTTON)
         .should('be.visible')
         .and('have.text', 'Correlations');
 
-      // TODO actual test
+      cy.log('should render all the correlations sections');
+
+      cy.get(CORRELATIONS_ANCESTRY_SECTION)
+        .should('be.visible')
+        .and('have.text', '1 alert related by ancestry');
+
+      cy.get(CORRELATIONS_SOURCE_SECTION)
+        .should('be.visible')
+        .and('have.text', '0 alerts related by source event');
+
+      cy.get(CORRELATIONS_SESSION_SECTION)
+        .should('be.visible')
+        .and('have.text', '1 alert related by session');
+
+      cy.get(CORRELATIONS_CASES_SECTION).should('be.visible').and('have.text', '0 related cases');
```

Comment:
if you take a look at [this Cypress test implementation](https://github.com/elastic/kibana/blob/main/x-pack/plugins/security_solution/cypress/e2e/investigations/alerts/expandable_flyout/alert_details_right_panel_overview_tab.cy.ts#LL250C26-L250C37), you could (if you want to) add the alert to a new case, which then should appear here